ambry-api/src/main/java/com/github/ambry/messageformat/BlobProperties.java

```diff
@@ -209,6 +227,16 @@ public String toString() {
     sb.append(", ").append("ContainerId=").append(getContainerId());
     sb.append(", ").append("IsEncrypted=").append(isEncrypted());
     sb.append(", ").append("externalAssetTag=").append(getExternalAssetTag());
+    if (getContentEncoding() != null) {
+      sb.append(", ").append("ContentEncoding=").append(getContentEncoding());
+    } else {
+      sb.append(", ").append("ContentEncoding=Null");
```

Comment:
I think this null handling case is not actually required, although I see that this pattern was used before in this method.

`sb.append("ContentEncoding=").append(null)` will work and return "ContentEncoding=null"

Reply:
Removed all the redundant part.